packages/default-theme/cms/elements/CmsElementCategorySidebarFilter.vue

```diff
@@ -69,7 +69,7 @@
         <div class="filters">
           <div v-for="filter in filters" :key="filter.name">
             <SfHeading class="filters__title" :level="4" :title="filter.name" />
-            <div v-if="filter && filter.options && filter.options.length">
+            <div v-if="filter && filter.options && filter.options.length" :class="{'filters__filter--color': filter && filter.name && filter.name === 'color'}">
```

Comment:
v-if checks if filter is not null, so we can safely make it shorter :)
```suggestion
            <div v-if="filter && filter.options && filter.options.length" :class="{'filters__filter--color': filter.name === 'color'}">
```

Reply:
Sure!